_docs/payout-details.md

```diff
@@ -443,6 +443,42 @@ Please note that the fields above are generally considered optional for senders
 
 ## XOF::Mobile
 
+### Ivory Coast
```

Comment:
Since the details we require are the same effectively for both CI and SN, so let's merge the two. We can add notes on what providers are supported in SN and IC respectively